src/geometries/PolyhedronGeometry.js

```diff
@@ -5,7 +5,7 @@ import { Vector2 } from '../math/Vector2.js';
 
 class PolyhedronGeometry extends BufferGeometry {
 
-	constructor( vertices, indices, radius = 1, detail = 0 ) {
+	constructor( vertices = [], indices = [], radius = 1, detail = 0 ) {
```

Comment:
I personally consider `PolyhedronGeometry` as an abstract class (so one should always derive from `PolyhedronGeometry`). Since the class is public and the doc page shows a code example with a direct usage, I've added two additional optional parameters to ensure an object creation without parameters is possible. Previously there was a runtime error. Now, an empty geometry is returned (which is fine from my point of view since there is no proper default for a polyhedron geometry).